lambdas/issuecredential/src/test/java/uk/gov/di/ipv/cri/drivingpermit/api/pact/IssueCredentialHandlerTest.java

```diff
@@ -0,0 +1,432 @@
+
+        PersonIdentityDateOfBirth birthDate = new PersonIdentityDateOfBirth();
+        birthDate.setValue(LocalDate.of(1962, 10, 11));
+
+        CanonicalAddress address = new CanonicalAddress();
+        address.setAddressCountry("GB");
+        address.setPostalCode("BS981TL");
+        address.setValidFrom(LocalDate.now().minusDays(1));
+
+        PersonIdentityItem personIdentityItem = new PersonIdentityItem();
+        personIdentityItem.setExpiryDate(
+                LocalDate.of(2030, 1, 1).toEpochSecond(LocalTime.now(), ZoneOffset.UTC));
+        personIdentityItem.setSessionId(sessionId);
+        personIdentityItem.setAddresses(List.of(address));
+        personIdentityItem.setNames(List.of(name));
+        personIdentityItem.setBirthDates(List.of(birthDate));
+
+        when(personIdentityDataStore.getItem(sessionId.toString())).thenReturn(personIdentityItem);
+
+        // SESSION HANDBACK
+        performAuthorizationCodeSet(sessionService, sessionId);
+        // SESSION HANDBACK
+
+        // ACCESS TOKEN GENERATION AND SETTING
+        SessionItem session = performAccessTokenSet(sessionService, sessionId);
+        // ACCESS TOKEN GENERATION AND SETTING
+
+        when(dataStore.getItemByIndex(SessionItem.ACCESS_TOKEN_INDEX, "Bearer dummyAccessToken"))
+                .thenReturn(List.of(session));
+    }
+
+    @State("dummyInvalidAccessToken is an invalid access token")
+    void invalidAccessTokenIsInvalidAccessToken() {}
+
+    @State("test-subject is a valid subject")
+    void jwtSubjectIsValid() {}
+
+    @State("dummyDrivingLicenceComponentId is a valid issuer")
+    void componentIdIsValidIssue() {}
+
+    @State("VC evidence activityHistoryScore is 1")
+    void vcHasDesiredActivityHistoryScore() {}
+
+    @State("VC has a CI of D02")
+    void vcHasDesiredCi() throws ParseException {
+        UUID sessionUUID =
+                sessionService
+                        .getSessionByAccessToken(
+                                AccessToken.parse(
+                                        "Bearer dummyAccessToken", AccessTokenType.BEARER))
+                        .getSessionId();
+        String sessionId = sessionUUID.toString();
+
+        DocumentCheckResultItem documentCheckResultItem = createBaseDocumentResultItem(sessionUUID);
+        documentCheckResultItem.setDocumentNumber("PARKE610112PBFGH");
+        documentCheckResultItem.setCheckMethod("data");
+        documentCheckResultItem.setIdentityCheckPolicy("published");
+        documentCheckResultItem.setActivityFrom("1982-05-23");
+        documentCheckResultItem.setValidityScore(0);
+        documentCheckResultItem.setIssuedBy("DVLA");
+        documentCheckResultItem.setIssueNumber("12");
+        documentCheckResultItem.setIssueDate("1982-05-23");
+        documentCheckResultItem.setContraIndicators(List.of("D02"));
+        documentCheckResultItem.setActivityHistoryScore(1);
```

Comment:
Should activity history score be 1 in this scenario?

Reply:
I believe this was the score provided by Dan for this scenario, do you believe it should be something else?

Reply:
Discussed with Dom, and updated to make this 0